server/etcdmain/etcd.go

```diff
@@ -57,6 +57,8 @@ func startEtcdOrProxyV2(args []string) {
 
 	err := cfg.parse(args[1:])
```

Comment:
I wonder why this error is only checked on line 70.

Reply:
If think the whole logic says: 

If we failed to parse the whole configuration, print the error using preferable the resolved logger from the config, but if does not exists, create a new temporary logger.

Reply:
Thank you!